sbnci/scripts/pfpvalidation.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+## Allows for different naming structure in cfg file
+export ref_sbndcode_ana_hist=$(echo ${ref_pfp_hist})
```

Comment:
Also here I think
`export ref_sbndcode_ana_hist=${ref_pfp_hist}`
should just work